Heffner_Daniel/Rails Section 2/upgraded_layout/app/views/dojo/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<% content_for :title do %>
+  Dojo Homepage
+<% end %>
+<% content_for :jquery_code do %>
+<script type='text/javascript'>
+ $(document).ready(function(){
+  $('form').submit(function(){
+   $.post($(this).attr('action'), $(this).serialize(),
+    function(data){
+     console.log(data);
+    });
+   return false;
+  });
+ });
+</script>
+<% end %>
+<% content_for :body do %>
```

Comment:
Usually content_for :body isn't really used, just a yield in the body so no content_for needed.